docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,142 @@
+  their preferred OAuth providers (e.g., GitHub, Google). Authenticated users are
+  redirected to JHub's homepage, where they select and launch the desired service.
+- As an external service managed by JupyterHub, JHub Apps service maintains full access to
+  JupyterHub’s endpoint API's: authentication, authorization, and access controls. As a result, users
+  still maintain the capability to visit Hub's built-in pages.
+- On the homepage, users can choose the specific framework to deploy as a single-user
+  server. JHub Apps then spawns the selected framework similarly to how JupyterLab is
+  launched. The service redirects users to their individual server instances, where they
+  can interact with their deployed application.
+
+## Looking Under the Hood
+
+When a user selects a framework, JHub Apps balances the given requests into two possible
+proxies, each one corresponding to a different deployment type:
+- For standard JupyterLab instances, the request is only managed by the default
+[jupyter-server-proxy](https://github.com/jupyterhub/jupyter-server-proxy), which
+launches the JupyterLab instance, under your usual `/lab` URL.
+- For other frameworks, an additional proxy layer is launched within the web-service, the
+[jhsingle-native-proxy](https://github.com/ideonate/jhsingle-native-proxy), which
+works in a similar fashion as the above, but is designed to remove the dependency on the
+jupyter notebook and jupyterlab services, while also allowing for a direct connection to the
+selected framework (webs service) during its lifecycle.
+
+Here’s a simplified explanation of the interaction between the two proxies:
+
+1. **Public Proxy (configurable-http-proxy):**
+   - Acts as a gatekeeper, for all receiving requests.
+   - This proxy decides where each request should go based on the user's information and
+     routes it accordingly.
+2. **Internal Proxies:**
+    - [jupyter-server-proxy](https://github.com/jupyterhub/jupyter-server-proxy):
+      - Jupyter Server Proxy lets you run arbitrary external processes (such as RStudio,
+        Shiny Server, Syncthing, PostgreSQL, Code Server, etc) alongside your notebook
+        server and provide authenticated web access to them using a path like /rstudio
+        next to others like `/lab`.
+    - [jhub-single-native](https://github.com/ideonate/jhsingle-native-proxy):
+      - Similar to the public proxy, it also dynamically updates the available route
+        table, it is, allows a public facing endpoint to be created for the selected
+        web-service (framework), though its main difference is that it is designed to be
+        a lightweight proxy, removing the dependency on the jupyter notebook and
+        jupyterlab services.
+
+### How does all Work Together?
+
+As an example, the diagram below illustrates the interaction between the two given
+proxies (http--jupyter-server-proxy or http--jhub-single-native), in a kubernetes
```

Comment:
```suggestion
proxies (`http--jupyter-server-proxy` or `http--jhub-single-native`), in a Kubernetes
```

Reply:
Just to double check - those are supposed to be double dashes, right? e.g. `http--`